### PR TITLE
Third Homework

### PR DESCRIPTION
**Pull request by Kalixone**

### CODE REVIEW

File: src/main/java/mate/academy/springbootintro/service/BookServiceImpl.java [hunk 1]
```diff
@@ -22,22 +24,39 @@ public class BookServiceImpl implements BookService {
...
                 .map(bookMapper::toDto)
                 .toList();
     }
 
     @Override
     public BookDto getBookById(Long id) {
-        Book book = bookRepository.getBookById(id).orElseThrow(
+        Book book = bookRepository.findById(id).orElseThrow(
                 () -> new EntityNotFoundException("Can't find book by id: " + id));
         return bookMapper.toDto(book);
```
Comment (nowickimj) [inline src/main/java/mate/academy/springbootintro/service/BookServiceImpl.java]: Multiple unnecessary new lines - your code doesn't look consistent if you sometimes put new line after return and sometimes you don't, in one method lines are separated and in the other they are not.

File: checkstyle.xml [hunk 2]
```diff
@@ -44,19 +44,6 @@
         </module>
         <module name="NeedBraces"/>
         <module name="LeftCurly"/>
-        <module name="RightCurly">
```
Comment (nowickimj) [inline checkstyle.xml]: Why did you remove this?

File: pom.xml [hunk 3]
```diff
@@ -22,7 +22,6 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-jpa</artifactId>
 		</dependency>
-
```
Comment (nowickimj) [inline pom.xml:25]: Unnecessary modification - do not apply changes to files not related to your task.

**Review by nowickimj — CHANGES_REQUESTED** [review]
minor changes requested

File: checkstyle.xml (hunk 2 above)
Comment (Kalixone) [reply checkstyle.xml]: I removed it - <module name="RightCurly"> because when I tried to build the project (mvn clean package) I got an error -  '}' at column 9 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks : if/else-if/else, to/in
hile or try/catch/finally). [RightCurlySame] it shows in Mapper, i tried to fix it, but i failed so i deleted it

Comment (nowickimj) [reply checkstyle.xml]: I see that your checkstyle plugin validates the compilation output as well. 
You can limit the scan only to source directories by modifying plugin's configuration in `pom.xml`:
```xml
<configuration>
	<sourceDirectories>
		<sourceDirectory>${project.build.sourceDirectory}</sourceDirectory>
		<sourceDirectory>${project.build.testSourceDirectory}</sourceDirectory>
	</sourceDirectories>
	<configLocation>${maven.checkstyle.plugin.configLocation}</configLocation>
	<consoleOutput>true</consoleOutput>
	<failsOnError>true</failsOnError>
	<linkXRef>false</linkXRef>
</configuration>
```

**Review by nowickimj — CHANGES_REQUESTED** [review]
Please fix the checkstyle plugin behavior by pointing the source directories:
```xml
	<sourceDirectories>
		<sourceDirectory>${project.build.sourceDirectory}</sourceDirectory>
		<sourceDirectory>${project.build.testSourceDirectory}</sourceDirectory>
	</sourceDirectories>
```

File: pom.xml (hunk 3 above)
Comment (nowickimj) [reply pom.xml:25]: Not fixed

File: src/main/java/mate/academy/springbootintro/SpringBootIntroApplication.java [hunk 4]
```diff
@@ -8,6 +8,5 @@ public class SpringBootIntroApplication {
 
 	public static void main(String[] args) {
 		SpringApplication.run(SpringBootIntroApplication.class, args);
-
```
Comment (nowickimj) [inline src/main/java/mate/academy/springbootintro/SpringBootIntroApplication.java]: Unnecessary change

File: checkstyle.xml (hunk 2 above)
Comment (nowickimj) [reply checkstyle.xml]: Please add changes to `pom.xml`

**Review by nowickimj — CHANGES_REQUESTED** [review]

**Review by nowickimj — APPROVED** [review]
